board/miyoo/main/apps/usb-host/usb-host.sh

```diff
@@ -1,3 +1,13 @@
 #!/bin/busybox sh
 echo host  > /sys/devices/platform/soc/1c13000.usb/musb-hdrc.1.auto/mode
 sleep 2
+
+modprobe evdev
+modprobe joydev
+modprobe analog
+modprobe xpad
+modprobe atkbd
+modprobe uinput
+modprobe psmouse
+modprobe synaptics_usb
+modprobe mousedev
```

Comment:
same as above, but maybe `rmmod` should be handled here by separate script (e.g. trigerred by gstreamer exec, which default to usb-mtd and remove all the rest of modules for speed)?

Reply:
yeah I forgot we need them only to not get triggered at start